pkg/operator/ceph/nfs/spec.go

```diff
@@ -66,6 +67,10 @@ func (r *ReconcileCephNFS) generateCephNFSService(nfs *cephv1.CephNFS, cfg daemo
 		svc.Spec.ClusterIP = v1.ClusterIPNone
 	}
 
+	if nfs.Spec.Server.LogLevel == "DEBUG" {
```

Comment:
NFS actually needs a specific command to enable debug logging instead of this. Inside the `daemonContainer()` method I believe you'll need to pass these args: `"-N", "NIV_DEBUG"` to the container that runs `ganesha.nfsd`. @varshar16 Is that correct?

Reply:
Yes. @subhamkrai you will need to make the change here: https://github.com/rook/rook/blob/dd53e75cf9c1d0a642be8cc22b549971e1c932f2/pkg/operator/ceph/nfs/spec.go#L204-L208

There are multiple log levels: 
https://github.com/nfs-ganesha/nfs-ganesha/blob/db29013c38ea7badd2be069fe51b127b1122b6a0/src/include/log.h#L54-L66

Reply:
Thanks for explaining. Done